sass/includes/_highlight-cards.scss

```diff
@@ -0,0 +1,76 @@
+.highlight-cards {
+    background-color: $color__yellow_6;
+    $root: &;
+    font-family: $font__supria-sans;
+
+    &__container {
+        padding: 3% 5%;
+
+        @media screen  and (min-width: map-get($grid-breakpoints, "sm")) {
+            padding: 3% 9%;
+            
+        }
+    }
+
+    &__link {
+        text-decoration: none !important;
```

Comment:
Note: I'm really not a fan of using !important, but the current targetting specificity of the `a` in typography means I need to use this or make multiple site wide changes.

Reply:
I agree, as a general rule we want to avoid the use of !important but in certain cases we can use it.